src/User/EmployeeList.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+
+const StyledEmployeesList = styled.ul`
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+`;
+const StyledEmployeesListItem = styled.li`
+  padding: 5px;
+`;
+const StyledEmployeesLink = styled(Link)`
+  display: inline-block;
+  text-decoration: none;
+  color: #7a517d;
+`;
+
+const EmployeeList = ({ employees }) => {
+  return (
+    <StyledEmployeesList className="employeeList">
```

Comment:
Is the class "employeeList" used anywhere, do you need it?

Reply:
yeah it's used in `ProfilePage` line 89